app/templates/base.html

```diff
@@ -5,51 +5,53 @@
 <!-- CSS CUSTOM -->
 <link rel="stylesheet" type="text/css" href="{% static 'css\css.css' %}">
 <html lang="en">
-<head>
-  <link rel="stylesheet" href="../../static/css/uf_table.css">
-  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.css"/>
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
-  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
-  <script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
-  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"></script>
-  <meta charset="UTF-8">
-  <title>{% block title %}{% endblock title %} - Upfronts</title>
-</head>
-<body>
-  <nav class="navbar navbar-expand-lg navbar-dark fixed-top color-navbar">
-    <div class="container-fluid">
-      <img src={% static "img/eventbrite.png" %} alt="eventbrite" width="120" height="40">
-      <a class="navbar-brand" href=""> Upfronts</a>
-      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
-            <span class="navbar-toggler-icon"></span>
-          </button>
-      <div class="collapse navbar-collapse" id="navbarResponsive">
-        <ul class="navbar-nav ml-auto">
-          <li class="nav-item active d-flex">
-            <a class="nav-link" href="{% url 'contracts' %}">Contracts
-                <span class="sr-only">(current)</span>
-            </a>
-          </li>
+  <head>
+    {% block head %}
+    <link rel="stylesheet" href="../../static/css/uf_table.css">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.css"/>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+    <script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"></script>
```

Comment:
I could suggest you start to use a  `defer` attribute for those scripts links. You are loading a lot of js stuff, usign `defer` you can unblock the page render. Check this article for more info. 
https://pegasaas.com/deferring-javascript-can-dramatically-speed-up-website/
https://javascript.info/script-async-defer